caput/tests/test_time.py

```diff
@@ -16,6 +16,23 @@
 from caput import time as ctime
 
 
+def setUpModule():
```

Comment:
Is this the only test that triggers the download? Maybe this function should be on a higher level as a helper available to tests.

Reply:
It originally was, but then I figured this was the only place that needed it, so just put it directly in here. I think if we ever place tests for the skyfield code in another module we'll need to do that, but I don't really see that happening.